src/RailWay/java/PageObjects/ChangePassWordPage.java

```diff
@@ -0,0 +1,58 @@
+package PageObjects;
+
+import Common.constant.Constant;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ChangePassWordPage extends GeneralPage {
+    //Locator
+    private final By tabChangePassWord = By.xpath("//a[@href='/Account/ChangePassword.cshtml']");
+    private final By txtCurrentPassWord = By.xpath("//li[@class='current-password']/input[@id='currentPassword']");
+    private final By txtNewPassWord = By.xpath("//li[@class='new-password']/input[@id='newPassword']");
+    private final By txtConfirmPassWord = By.xpath("//li[@class='confirm-password']/input[@id='confirmPassword']");
+    private final By btnChangePassWord = By.xpath("//p[@class='form-actions']/input[@type='submit']");
+    private final By paSuccessChangePass = By.xpath("//form[@id='ChangePW']//p[@class='message success']");
```

Comment:
What does pa stand for? Please update refer this link
https://stackoverflow.com/questions/45028747/suggested-naming-conventions-for-selenium-identifiers

Reply:
i have update from pa to lbl